src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -0,0 +1,53 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.dto.CartDto;
+import com.kodilla.ecommercee.domain.dto.CartItemDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping(value = "/carts")
+public class CartController {
+    
+    @PostMapping
+    public ResponseEntity<Void> createCart() {
+        List<CartItemDto> cart = new ArrayList<>();
+        //post to repository//
+        return ResponseEntity.ok().build();
+    }
+    
+    @GetMapping("/{cartId}")
+    public ResponseEntity<CartDto> getCart(Long cartId) {
+        //get from repository//
+        //map to Dto//
+        CartDto currentCard = new CartDto(cartId, new ArrayList<>());
+        return ResponseEntity.ok(currentCard);
+    }
+
+    @PostMapping(value = "/addItem")
+    public ResponseEntity<Void> addCartItem(@RequestBody CartItemDto cartItemDto) {
+        //post item into Cart//
+        return ResponseEntity.ok().build();
+    }
```

Comment:
We add item, but we don't know where we add it. It could be resolve by PathVariable or RequestParam. My proposition:

 @PostMapping(value = "/{cartId}")
    public ResponseEntity<Void> addCartItem(@PathVariable Long cartId, @ReqestBody CartItemDto cartItemDto) {
        //post item into Cart//
        return ResponseEntity.ok().build();
    }

Reply:
Right